class raise_3:



    def raise3(self):
        for i in range(1,11):
           #강제적으로 에러  우리가 만든 메세지 출력
            raise Exception('에러가 났습니다!!!')



ra=raise_3()
ra.raise3()